src/Proto.Actor/Process.cs

```diff
@@ -40,16 +45,19 @@ internal bool IsDead
 
         protected internal override void SendUserMessage(PID pid, object message)
         {
+            if (IsDead) return;
             Mailbox.PostUserMessage(message);
         }
 
         protected internal override void SendSystemMessage(PID pid, object message)
         {
+            if (IsDead) return;

```

Comment:
@rogeralsing  Here we got a bit problem. Will there be a lot of performance hit if we check IsDead here?

The reason I'm doing this is I want to set Mailbox to null when stopping, otherwise all the messages in that Mailbox will not be garbage collected and causing memory leak.

Another way is to clear the queue in Mailbox.
What you think?